packages/cli/src/middlewares/listQuery/dtos/workflow.filter.dto.ts

```diff
@@ -0,0 +1,40 @@
+import { isObjectLiteral } from '@/utils';
+import { IsOptional, IsString, IsBoolean, IsArray, validate } from 'class-validator';
+import { instanceToPlain } from 'class-transformer';
+
+export class WorkflowFilter {
+	@IsString()
+	@IsOptional()
+	name: string;
+
+	@IsBoolean()
+	@IsOptional()
+	active?: boolean;
+
+	@IsArray()
+	@IsString({ each: true })
+	@IsOptional()
+	tags?: string[];
```

Comment:
Amazing code! Filtering by tags doesn't seem to work tho. Could you please check?

Everything else is just perfect 💯

Reply:
Thanks :)

Can you provide more details? I see it working [in Insomnia](https://share.cleanshot.com/H282wS6D) and the [workflow filter test](https://github.com/n8n-io/n8n/pull/6892/files#diff-e5c921d2cb1e160cd0a1d479024a9194bee51289db61cbcd3b834e6d6be3c9e9R168) is passing.